### PR TITLE
chore: prepare v0.2.0 CHANGELOG

### PR DESCRIPTION
**Pull request by nikku**

### CODE REVIEW

**Comment by nikku** [comment]
@philippfromme Can be squashed and merged if appropriate.

File: packages/form-js/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,29 @@ All notable changes to [form-js](https://github.com/bpmn-io/form-js) are documen
...
+* `FIX`: correct submission of empty fields ([#54](https://github.com/bpmn-io/form-js/issues/54))
+* `FIX`: correct submission of disabled fields ([#6](https://github.com/bpmn-io/form-js/issues/6))
+
+### Editor
+
+* `FEAT`: add undo and redo ([#45](https://github.com/bpmn-io/form-js/issues/45))
+
+### Breaking Changes
+
+* Form viewer and editor import are now asynchronous. Creating either via `#createForm` and `#createFormEditor` must be awaited.
```
Comment (philippfromme) [inline packages/form-js/CHANGELOG.md:30]: Edtior import is still synchronous. It's what https://github.com/bpmn-io/form-js/issues/82 is about.

Comment (nikku) [reply packages/form-js/CHANGELOG.md:30]: Idea: Make it async, fake it async or pretend it's async.

Comment (philippfromme) [reply packages/form-js/CHANGELOG.md:30]: Pull Request: https://github.com/bpmn-io/form-js/pull/91

**Comment by philippfromme** [comment]
I'll add the remaining changes we just merged.

**Review by philippfromme — APPROVED** [review]
🚢 💨

**Pull request merged**